app/src/main/res/layout/activity_launch_screen.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".LaunchScreenActivity">
+
+    <TextView
+        android:id="@+id/txt_sign"
```

Comment:
Views in this layout should be constrained to start and end of the screen and have correct margin set